### DISCUSSION TITLE
Change search results display to show on top + easier launch method

### DISCUSSION THREAD

**Discussion by whmxtra** [Flow-Launcher/Flow.Launcher#3074] [Ideas]
There doesn't seem to be any option to choose whether Flow shows the search results on the top or bottom, they are just default on the bottom.

Problem is if you want to put the Flow search box at the bottom of your screen closer to the taskbar then the results go right off the bottom of the screen. There's no way to make the results appear on top of the search box. Technically it should automatically detect the location of the search box and if there's not enough room to display under the box it should default to displaying on the top of it.

This is really one of the few things that are stopping me from using Flow on a daily basis.

Another issue is launching. Sometimes it's a pain to have to hit the hot key, it's just easier to click the windows search icon. Is it possible to also add an option to have the search box display, say, when you mouse to a screen corner or something?

**Comment by jjw24** [comment]
Similar to https://github.com/Flow-Launcher/Flow.Launcher/issues/1349

**Comment by whmxtra** [comment]
So is this actually going to be implemented at some point?
